outputs.nix

```diff
@@ -0,0 +1,49 @@
+{
+  std,
+  self,
+  nix-helm,
+  ...
+} @ inputs:
+inputs.flake-parts.lib.mkFlake {inherit inputs;} {
```

Comment:
But this was moved into `flake.nix`, then this file can be deleted?

Reply:
Yes, it can.